pipelinewise/cli/__init__.py

```diff
@@ -142,6 +144,59 @@ def __disable_profiler(
         profiler.clear()
 
 
+def _validate_command_specific_arguments(args):
+    # Command specific argument validations
+    if args.command == 'init' and args.name == '*':
+        raise CommandSpecificArgumentsException('You must specify a project name using the argument --name')
+
+    if args.command in ['discover_tap', 'test_tap_connection', 'run_tap', 'stop_tap']:
+        if args.tap == '*':
+            raise CommandSpecificArgumentsException('You must specify a source name using the argument --tap')
+        if args.target == '*':
+            raise CommandSpecificArgumentsException('You must specify a destination name using the argument --target')
+
+    if args.command == 'sync_tables':
+        if args.tap == '*':
+            raise CommandSpecificArgumentsException('You must specify a source name using the argument --tap')
+        if args.target == '*':
+            raise CommandSpecificArgumentsException('You must specify a destination name using the argument --target')
```

Comment:
the `if args.command == 'sync_tables':` can be merged with the first if.

Reply:
- [ ] Merge if statements